src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -805,6 +821,101 @@ def _resolve_rate(
             alias,
         )
 
+    def _resolve_regression_score(
+        self, args: Mapping[str, str | Column | SelectType | int | float], alias: str | None = None
+    ) -> SelectType:
+        return Function(
+            "minus",
+            [
+                Function(
+                    "multiply",
+                    [
+                        self._resolve_average_cond(args, None, "greater"),
```

Comment:
If alias is unused lets just omit the parameter entirely

Reply:
These are going to be exposed separately to show a breakdown of what's regressed across a breakpoint, but since they aren't used in this current PR I'll remove these args.

Reply:
Ah! if that's the case it would've been fine to keep it, my bad!